qt/aqt/browser/find_and_replace.py

```diff
@@ -33,19 +33,36 @@ class FindAndReplaceDialog(QDialog):
     COMBO_NAME = "BrowserFindAndReplace"
 
     def __init__(
-        self, parent: QWidget, *, mw: AnkiQt, note_ids: Sequence[NoteId]
+        self,
+        parent: QWidget,
+        *,
+        mw: AnkiQt,
+        note_ids: Sequence[NoteId],
+        find: Optional[str] = None,
+        replace: Optional[str] = None,
+        field: Optional[Union[str, bool]] = None,
     ) -> None:
+        """
+        If 'field' is True, preset 'All fields'; if False, 'Tags'; if a string,
+        the field name, adding it to the list if necessary.
+        """
```

Comment:
If `field` is passed (via the sidebar), the assumption is that the caller wants to modify every note with that field/tag, compared to only the selected notes, when `field` isn't passed (via the menubar/table context menu).
Since the "Fields" combobox is populated according to the passed `note_ids`, it may not contain the passed `field`. But since we assume that the user wants to modify the whole collection and they've already specified a field name, it makes sense to allow it.

Reply:
Ah, I see. It's not really useful to give users the ability to select a different field there when they're triggering it from the sidebar, but I see that's awkward to change due to the assumptions about all fields/tags being idx 0 and 1 there. We could potentially skip the QueryOp call and just call _show([field]) to avoid the field name lookups, but not sure if that's worth it or not.

Reply:
I think that even makes thing a little bit simpler, so I'll do that.